templates/templated_email/account/account_delete.email

```diff
@@ -5,9 +5,8 @@
 {% endblock %}
 
 {% block plain %}
+{% trans "Hi!" context "Standard e-mail greeting" %}
 {% blocktrans context "Account delete reset e-mail text" %}
-Hi!
-
 You're receiving this e-mail because you or someone else has requested a deletion of your user account at {{ domain }}.
```

Comment:
Since we're going to break translation (there's no good way around it), I think that's a good opportunity to review the email copy with @timcla.

Reply:
Good idea 👍

Reply:
@Kwaidan00 will gather all of our email contents in a single file and we'll ask @timcla for review and improvements. Once we have it done, we'll update the templates.